xbmc/utils/ActorProtocol.cpp

```diff
@@ -168,7 +168,7 @@ bool Protocol::SendOutMessage(int signal, CPayloadWrapBase *payload, Message *ou
   return true;
 }
 
-bool Protocol::SendInMessage(int signal, void *data /* = NULL */, size_t size /* = 0 */, Message *outMsg /* = NULL */)
+bool Protocol::SendInMessage(int signal, const void *data /* = NULL */, size_t size /* = 0 */, Message *outMsg /* = NULL */)
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting

> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types